lib/tizen_sdk.dart

```diff
@@ -271,7 +276,7 @@ class TizenSdk {
     String type = arch == 'x86' ? 'emulator' : 'device';
     if (arch == 'arm64') {
       // The arm64 build is only supported by iot-headed-6.0+ rootstraps.
-      if (profile != 'iot-headed') {
+      if (profile != 'tizen' && profile != 'iot-headed') {
         _logger.printError(
             'The arm64 build is not supported by the $profile profile.');
         profile = 'iot-headed';
```

Comment:
You may move lines 263-266 after line 288 so that the profile is not overwritten at line 282, or just remove the check (lines 279-283) for simplicity.

Reply:
I moved if-condition to later. This change is no longer needed.

Reply:
You may still need the check. If the user specified the profile as `tizen`, the tool will emit the error. Yes, it's quite complicated and unreadable. This is why I recommended removing the part.

Reply:
I modified it. thanks